app/models/policy_group.rb

```diff
@@ -65,9 +65,7 @@ def base_path
   end
 
   def public_path(options = {})
-    locale = options[:locale] || "en"
-
-    append_url_options(base_path, options, locale:)
+    append_url_options(base_path, options, locale: I18n.default_locale)
```

Comment:
I think actually now I want to change this and just add it in as a keyword arg so that everything is uniform (makes it easier when we're doing dynamic invocations (😢) of stuff too